.github/workflows/web.yml

```diff
@@ -51,6 +51,16 @@ env:
   widgets_directory: centreon/www/widgets
 
 jobs:
+  set-pr-environment:
+    uses: ./.github/workflows/set-pr-environment.yml
+
+  get-version:
```

Comment:
@tuntoja & @kduret , do you see any issue in moving get-version before the over actions ?